src/Extension/Input.php

```diff
@@ -136,7 +136,7 @@ public function formGroup(string $label, Field $field, string $inputHtml) : stri
         $inputDocument->loadHTML($inputHtml);
 
         foreach ($inputDocument->getElementsByTagName('body')->item(0)->childNodes as $childNode) {
-            $inputNode = $document->importNode($childNode);
+            $inputNode = $document->importNode($childNode, true);
```

Comment:
This fixes all the cases I saw. Options inside selects and text inside textareas.